docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -515,6 +518,25 @@ Equivalent to the Podman `--health-cmd` option.
 Set an interval for the healthchecks. An interval of disable results in no automatic timer setup.
 Equivalent to the Podman `--health-interval` option.
 
+### `HealthLogDestination=`
+
+Set the destination of the HealthCheck log. Directory path or events_logger (none use container state file)
+Equivalent to the Podman `--health-log-destination` option.
+
+* `none`: (default) HealthCheck logs are stored in overlay containers. (For example: `./run/containers/storage/overlay-containers/<container-ID>/healthcheck.log`)
+* `directory`: creates a log file named `<container-ID>-healthcheck.log` with HealthCheck logs in the specified directory.
+* `journal`: The log will be written to the journal. A log with HealthCheck logs will also be written to the default path.
+
+### `HealthMaxLogCount=`
+
+Set maximum number of attempts in the HealthCheck log file. ('0' value means an infinite number of attempts in the log file)
+Equivalent to the Podman `--Health-max-log-count` option.
```

Comment:
What's the default here?

Reply:
5 attempts

Reply:
Please put these in the manpage, make it clear to users what the expectation is.

Reply:
I have added default values to man pages.